src/components/CountrySelector/style.scss

```diff
@@ -100,9 +101,9 @@ $_sdk-option-padding: 8 * $unit-small;
 $_sdk-option-border-bottom: 0;
 
 %_sdk-caret-icon {
```

Comment:
Why is it called `sdk-caret-icon` if it's not an icon 🤔

Reply:
Sorry, I meant it's not a caret icon ^. It' a magnifying glass, isn't it?

Reply:
True. On thinking more about it the most appropriate name for it is `%_sdk-search-icon`